### ISSUE TITLE
Unable to system-upgrade

### ISSUE THREAD

**Issue by denysvitali** [rpm-software-management/dnf-plugins-extras#134]
### dnf
`dnf-3.6.1-1.fc29.noarch`
### dnf plugins
`dnf-plugins-core-3.0.4-1.fc29.noarch

## Running on
Fedora 29

## Stacktrace

```
[dvitali@pixelc ~]$ sudo dnf system-upgrade download --refresh --releasever=rawhide
Before you continue ensure that your system is fully upgraded by running "dnf --refresh upgrade". Do you want to continue [y/N]: y
Traceback (most recent call last):
  File "/usr/bin/dnf", line 58, in <module>
    main.user_main(sys.argv[1:], exit_code=True)
  File "/usr/lib/python3.7/site-packages/dnf/cli/main.py", line 179, in user_main
    errcode = main(args)
  File "/usr/lib/python3.7/site-packages/dnf/cli/main.py", line 64, in main
    return _main(base, args, cli_class, option_parser_class)
  File "/usr/lib/python3.7/site-packages/dnf/cli/main.py", line 95, in _main
    cli.configure(list(map(ucd, args)), option_parser())
  File "/usr/lib/python3.7/site-packages/dnf/cli/cli.py", line 910, in configure
    self.command.configure()
  File "/usr/lib/python3.7/site-packages/dnf-plugins/system_upgrade.py", line 318, in configure
    self._call_sub("configure")
  File "/usr/lib/python3.7/site-packages/dnf-plugins/system_upgrade.py", line 330, in _call_sub
    subfunc()
  File "/usr/lib/python3.7/site-packages/dnf-plugins/system_upgrade.py", line 375, in configure_download
    self.base.conf.tsflags += ["test"]
TypeError: can only concatenate tuple (not "list") to tuple
```  
  
[This is the line that the stacktrace is pointing to](https://github.com/rpm-software-management/dnf-plugins-extras/blob/master/plugins/system_upgrade.py#L375)

**Comment by denysvitali** [comment]
Apparently `self.base.conf.tsflags` content is `()` and its type is `<class 'tuple'>`

**Comment by j-mracek** [comment]
I belive that the issue is resolved with dnf-plugins-core--4.0.1+
